Adds Swift code snippets

AmitaiB opened the pull request:
## SDK documentation PR
### What does this Pull Request do?
Adds the Swift equivalent code snippets next to the Objective-C code snippets. This is an extension of the already vetted and approved docs changes that authorized me to add to Objective-C code snippets their Swift equivalents.

### Why is this Pull Request needed?
To comply with our website's dual Swift/ObjC compatibility.

#### Pay Special Attention To:
In the migration guide, added Swift column, but not strictly a code snippet.


Do not forget to tag @kcorneli201 and at least one other reviewer.

kcorneli201 committed:
Merge pull request #44 from jwplayer/develop

Publishing documentation for 4/15/19

AmitaiB committed:
Adds Swift snippet

This is an extension of the already vetted and approved docs changes that authorized me to add to Objective-C code snippets their Swift equivalents.

kcorneli201 reviewed (approved):
@AmitaiB LGTM. But, please merge to develop first. Then, push to master. I want to keep both branches in sync.

@karimMourra / @henryhlee Do you see anything amiss in the code? Here is the page on staging: https://staging-developer.jwplayer.com/sdk/ios/docs/developer-guide/casting/airplay/

AmitaiB committed:
Separates ObjC and Swift snippets

According to the new template that Design...designed.

AmitaiB committed:
Adds Swift snippets

Also adds 2 words for clarity, on lines 49 and 58.

AmitaiB committed:
Add Swift snippet to ad schedules

AmitaiB committed:
Adds Swift snippet to Ad Waterfalling

AmitaiB committed:
Adds Swift snippet to Non-linear ads

AmitaiB committed:
Adds Swift snippet to Google IMA

AmitaiB committed:
Adds Swift snippet to Freewheel integration page

Removes illegal special characters (smart quotes, replaced with unicode actual quotes).
Renames all "conf" to "config" for parity with the rest of the code snippets in the developer's guide.

AmitaiB committed:
Adds Swift snippet to Ad Rules page

AmitaiB committed:
Adds Swift snippet with protocol analogue

I add an explanatory comment in the snippet to make it clear that the Swift protocol presented is for illustration purposes only.

AmitaiB committed:
Adds Swift enums to the Name Changes table
